etc/examples/pytorch-model/mnist.py

```diff
@@ -171,3 +172,14 @@ def test():
 for epoch in range(1, args.epochs + 1):
     train(epoch)
     test()
+
+    
+os.makedirs(os.environ["RESULT_DIR"]+'/model')    
```

Comment:
@Tomcli where will this save the model - local os?

Reply:
It's stored under the `RESULT_DIR` path. I think here is just a work around solution to package the model in `tar.gz` format.

Reply:
I just copied what was in the tf-model example for storing the model.